### PR TITLE
Fix e2e in the CI for AWS

### PR DESCRIPTION
**Pull request by tarekziade**
Make sure all e2e test work in buildkite

### CODE REVIEW

**Comment by tarekziade** [comment]
I got `Amazon` working

File: connectors/sources/s3.py [hunk 1]
```diff
@@ -26,9 +26,11 @@
...
 DEFAULT_PAGE_SIZE = 100
 DEFAULT_CONTENT_EXTRACTION = True
-ENDPOINT_URL = os.getenv(
-    key="ENDPOINT_URL", default=None
-)  # For end to end tests set this environment variable with target host.
+
+if "AWS_ENDPOINT_URL" in os.environ:
+    AWS_ENDPOINT = f"{os.environ['AWS_ENDPOINT_URL']}:{os.environ['AWS_PORT']}"
+else:
+    AWS_ENDPOINT = None
```
Comment (artem-shelkovnikov) [inline connectors/sources/s3.py:33]: What does the `None` case mean?

Comment (artem-shelkovnikov) [reply connectors/sources/s3.py:33]: I guess when it's null it uses real AWS?

**Review by artem-shelkovnikov — APPROVED** [review]
LGTM! Left a small question

File: connectors/sources/s3.py (hunk 1 above)
Comment (tarekziade) [reply connectors/sources/s3.py:33]: yes. see https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html#boto3.session.Session.client

**Pull request merged**